gulpfile.js

```diff
@@ -192,33 +218,26 @@ gulp.task('styles-grid', function() {
     .pipe($.size({title: 'styles-grid'}));
 });
 
+// Build with Google's Closure Compiler.
+// To run this step, run "bower install" within MDL to fetch the compiler jar,
+// note that Java 1.7+ is required. This is an optional step.
+gulp.task('closure', function() {
+  return gulp.src(SOURCES)
+    .pipe(closureCompiler({
+      compilerPath: 'bower_components/closure-compiler/compiler.jar',
+      fileName: './dist/mdl.closure.min.js',
```

Comment:
That doesn't seem to match my experience when I run the task locally. I get a `dist/material.closure.min.js` file created and then promptly deleted. However, if I specify the `fileName` without any paths (i.e., just `'material.closure.min.js`), with no further changes to compiler flags, and then pipe it to the destination (`.pipe(gulp.dest('./dist'))`) everything works and there are no piping issues (presumably because the file gets created locally and then moved?).

Could you run an `npm install` to make sure you're on the latest version of all our dependencies?